server/routes/appointments/create-and-edit/handlers/dateAndTime.ts

```diff
@@ -62,6 +62,9 @@ export default class DateAndTimeRoutes {
     appointmentJourney.retrospective = YesNo.NO
 
     if (appointmentJourney.mode === AppointmentJourneyMode.COPY) {
+      if (appointmentJourney.repeat === YesNo.YES) {
+        return res.redirectOrReturn('copy-series')
```

Comment:
If copying a series and new series is in the future then ask if the user wants to copy the series or the single appointment